### PR TITLE
[fix][broker] Fix Replicated Topic unload bug when ExtensibleLoadManager is enabled

### PR DESCRIPTION
**Pull request by heesung-sn**
### Motivation

Noticed that Replicated Topic is not unloaded upon setNamespaceReplicationClusters when ExtensibleLoadManager is enabled. 

### Modifications

- Fix unloading bundles upon namespace policy update. (isNamespaceBundleOwned didn't check the bundle ownership properly when ExtensibleLoadManager is enabled.)
- Do not replicate ExtensibleLoadManager internal topics (bundle state, broker and bundle load data topics) because they are only used for the local cluster.
- Copy ReplicatorGlobalNSTest for ExtensibleLoadManager.

### Verifying this change

- [x] Make sure that the change passes the CI checks.

### Does this pull request potentially affect one of the following parts:

*If the box was checked, please highlight the changes*

- [ ] Dependencies (add or upgrade a dependency)
- [ ] The public API
- [ ] The schema
- [ ] The default values of configurations
- [ ] The threading model
- [ ] The binary protocol
- [ ] The REST endpoints
- [ ] The admin CLI options
- [ ] The metrics
- [ ] Anything that affects deployment

### Documentation

- [ ] `doc` 
- [ ] `doc-required` 
- [x] `doc-not-needed` 
- [ ] `doc-complete` 

### Matching PR in forked repository

PR in forked repository:

### CODE REVIEW

**Review by Technoboy- — APPROVED** [review]

**Review by Demogorgon314 — APPROVED** [review]

**Review by Demogorgon314 — COMMENTED** [review]
Can we also check the other place such as `producer.close()`, looks like it may also get stuck forever.

**Comment by heesung-sn** [comment]
> Can we also check the other place such as producer.close(), looks like it may also get stuck forever.

I understand the producer.flush might get stuck if no leader broker is found in the cluster, but actually, I wonder how producer.close() can get stuck.

**Review by Demogorgon314 — APPROVED** [review]

File: pulsar-broker/src/test/java/org/apache/pulsar/broker/service/ReplicatorGlobalNSTest.java [hunk 1]
```diff
@@ -41,6 +45,18 @@
 public class ReplicatorGlobalNSTest extends ReplicatorTestBase {
 
     protected String methodName;
+    @DataProvider(name = "loadManagerClassName")
+    public static Object[][] loadManagerClassName() {
+        return new Object[][]{
+                {ModularLoadManagerImpl.class.getName()},
+                {ExtensibleLoadManagerImpl.class.getName()}
```
Comment (lhotari) [inline pulsar-broker/src/test/java/org/apache/pulsar/broker/service/ReplicatorGlobalNSTest.java:52]: @heesung-sn I think that this is triggering a problem in tests.

Comment (heesung-sn) [reply pulsar-broker/src/test/java/org/apache/pulsar/broker/service/ReplicatorGlobalNSTest.java:52]: Sure. We can comment out this line if this is problematic.

**Pull request merged**